src/ucp/proto/proto_debug.c

```diff
@@ -382,7 +385,7 @@ void ucp_proto_select_param_str(const ucp_proto_select_param_t *select_param,
         [ucs_ilog2(UCP_PROTO_SELECT_OP_FLAG_AM_RNDV)]  = "rndv"
     };
     unsigned op_flags                    = select_param->op_id_flags &
-                                           (UCP_PROTO_SELECT_OP_FLAGS_BASE - 1);
+                                           op_flags_bits;
```

Comment:
maybe `& ~(UCP_PROTO_SELECT_OP_FLAGS_BASE - 1)` ?
or add helper function similar to ucp_proto_select_op_id - ucp_proto_select_op_flags

Reply:
Added.
+ updated code related to processing of op_flags.